### PR TITLE
API updates to workspaces testing data

### PR DESCRIPTION
**Pull request by nathanstitt**
Adds an admin panel for explicitly setting study data and replies with that if it's available before generating it's own random data

### CODE REVIEW

File: backend/spec/requests/api/v1/researcher/studies_spec.rb
Comment (chrisbendel) [inline backend/spec/requests/api/v1/researcher/studies_spec.rb:1]: some odd formatting in this file

File: backend/lib/tasks/populate_study_researcher_data.rake
[the remark this answers is not in the document]
Comment (nathanstitt) [reply backend/lib/tasks/populate_study_researcher_data.rake:1]: oh did you add that?  rubocop didn't like an empty file so I'd removed it

File: backend/config/secrets.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ development:
   qualtrics_api_key: 000000
   qualtrics_api_url: https://iad1.qualtrics.com/API/v3
   qualtrics_group_library: 'GR_7R5NeLAPtj22K46'
+  enclave_api_key: ec_1bG346BDBGEsG5Dck8H6sAiRBJ7Vt52Vwcno
```
Comment (chrisbendel) [inline backend/config/secrets.yml:8]: should these api keys be env-ified?

Comment (nathanstitt) [reply backend/config/secrets.yml:8]: it isn't a "real" api key, it's just for local testing, note that it's only in the dev & test blocks.  

in production it does read from ENV

File: frontend/src/screens/admin/grid.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import { BoxProps } from 'boxible'
+
+export const Grid = styled.div({
+    display: 'grid',
+    height: '100vh',
+    width: '100vw',
+    gridTemplateColumns: 'minmax(auto, 200px) 1fr',
+    gridTemplateRows: 'auto 1fr auto',
+    overflow: 'hidden',
+    gridTemplateAreas: `
```
Comment (chrisbendel) [inline frontend/src/screens/admin/grid.tsx:11]: This is super cool, i didn't know this was a thing. read up on it a little more and wonder if down the line we could build out the whole site's structure like this!

File: backend/lib/tasks/populate_study_researcher_data.rake
[the remark this answers is not in the document]
Comment (chrisbendel) [reply backend/lib/tasks/populate_study_researcher_data.rake:1]: actually i dont think we'll need it, good to keep it out! 👍

**Pull request merged**